sdk/test/Services/DynamoDBv2/IntegrationTests/DataModelTests.cs

```diff
@@ -158,6 +157,32 @@ public void TestContext_DisableFetchingTableMetadata_DateTimeAsHashKey()
             Assert.AreEqual(employee.Age, storedEmployee.Age);
         }
 
+        /// <summary>
+        /// Tests that disabling fetching table metadata works with a key that has a property converter.
+        /// </summary>
+        [TestMethod]
+        [TestCategory("DynamoDBv2")]
+        public async Task TestContext_DisableFetchingTableMetadata_KeyWithPropertyConverter()
+        {
+            TableCache.Clear();
+            CleanupTables();
+            TableCache.Clear();
+
+            CreateContext(DynamoDBEntryConversion.V2, true, true);
+
+            var employee = new PropertyConverterEmployee
+            {
+                Name = Status.Active,
+                CreationTime = EpochDate,
+            };
+
+            await Context.SaveAsync(employee);
+            var storedEmployee = await Context.LoadAsync<PropertyConverterEmployee>(employee.CreationTime, employee.Name);
```

Comment:
These tests are still included in `AWSSDK.IntegrationTests.DynamoDBv2.Net35`, which doesn't have `Task` or the async methods on `Context`.

Can you adjust to `public void` and use the sync `Save` and `Load` methods instead?

Reply:
done